grid-client/workloads/vm.go

```diff
@@ -17,28 +18,31 @@ var ErrInvalidInput = errors.New("invalid input")
 
 // VM is a virtual machine struct
 type VM struct {
-	Name          string            `json:"name"`
-	Flist         string            `json:"flist"`
-	FlistChecksum string            `json:"flist_checksum"`
-	PublicIP      bool              `json:"publicip"`
-	PublicIP6     bool              `json:"publicip6"`
-	Planetary     bool              `json:"planetary"`
-	Corex         bool              `json:"corex"` //TODO: Is it works ??
-	ComputedIP    string            `json:"computedip"`
-	ComputedIP6   string            `json:"computedip6"`
-	YggIP         string            `json:"ygg_ip"`
-	IP            string            `json:"ip"`
-	Description   string            `json:"description"`
-	GPUs          []zos.GPU         `json:"gpus"`
-	CPU           int               `json:"cpu"`
-	Memory        int               `json:"memory"`
-	RootfsSize    int               `json:"rootfs_size"`
-	Entrypoint    string            `json:"entrypoint"`
-	Mounts        []Mount           `json:"mounts"`
-	Zlogs         []Zlog            `json:"zlogs"`
-	EnvVars       map[string]string `json:"env_vars"`
-	NetworkName   string            `json:"network_name"`
-	ConsoleURL    string            `json:"console_url"`
+	Name          string `json:"name"`
+	Flist         string `json:"flist"`
+	FlistChecksum string `json:"flist_checksum"`
+	PublicIP      bool   `json:"publicip"`
+	PublicIP6     bool   `json:"publicip6"`
+	Planetary     bool   `json:"planetary"`
+	Corex         bool   `json:"corex"` //TODO: Is it works ??
+	ComputedIP    string `json:"computedip"`
+	ComputedIP6   string `json:"computedip6"`
+	PlanetaryIP   string `json:"planetary_ip"`
+	IP            string `json:"ip"`
+	// used to get the same mycelium ip for the vm. if not set and planetary is used
+	// it will fallback to yggdrasil.
+	MyceliumSeed []byte            `json:"mycelium_seed"`
```

Comment:
MyceliumSeed is a confusing name in this context, should be changed to MyceliumIPSeed or MyceliumIPSuffix or something 

Notice that in ZOS while it's called just Seed it's encapsulated into a struct called MyceliumIP